ror/.rubocop.yml

```diff
@@ -1,4 +1,6 @@
 AllCops:
+  EnabledByDefault: true
```

Comment:
@MauricioRobayo @Dandush03 thank for your research on that. ❤️ 

According to the test repo mentioned by @MauricioRobayo we need those cops to be listed in rubocop config file:


```
Lint/RaiseException:
  Enabled: false
Lint/StructNewOverride:
  Enabled: false
Style/HashEachMethods:
  Enabled: false
Style/HashTransformKeys:
  Enabled: false
Style/HashTransformValues:
  Enabled: false
```

Can we please explicitly add those cops instead of enabling all by default and dealing with the ones that we obviously do not want to enable?

Thanks to that we have a more clear image in our Rubocop config - you can see what is enabled at first sight.
And controlling this file will be easier.